test.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const ecpect = require('chai');
+const readFile = require('./lib/readWrite.js');
+
+describe('readFile', function(){
+ it('should ');
```

Comment:
This test is a bit incomplete. Check [this](https://mochajs.org/#getting-started) for a complete example to compare to.